wiki/Announcements/en.md

```diff
@@ -0,0 +1,10 @@
+# Announcements
```

Comment:
this is an index page, it should be obvious enough as to what each page leads off to... 

...

Do you really still think I should add them (even after the point I tried making above)?